pkg/ebpf/btf.go

```diff
@@ -174,33 +214,33 @@ func (b *orderedBTFLoader) embeddedPath() (string, error) {
 
 var kernelVersionPatterns = []struct {
 	pattern   *regexp.Regexp
-	platforms []string
+	platforms []btfPlatform
 }{
-	{regexp.MustCompile(`\.amzn[1-2]\.`), []string{"amazon"}},
-	{regexp.MustCompile(`\.el7\.`), []string{"redhat", "centos", "oracle"}},
-	{regexp.MustCompile(`\.el8(_\d)?\.`), []string{"redhat", "centos", "oracle"}},
-	{regexp.MustCompile(`\.el[7-8]uek\.`), []string{"oracle"}},
-	{regexp.MustCompile(`\.deb10\.`), []string{"debian"}},
-	{regexp.MustCompile(`\.fc\d{2}\.`), []string{"fedora"}},
+	{regexp.MustCompile(`\.amzn[1-2]\.`), []btfPlatform{platformAmazon}},
+	{regexp.MustCompile(`\.el7\.`), []btfPlatform{platformRedhat, platformCentOS, platformOracle}},
+	{regexp.MustCompile(`\.el8(_\d)?\.`), []btfPlatform{platformRedhat, platformCentOS, platformOracle}},
+	{regexp.MustCompile(`\.el[7-8]uek\.`), []btfPlatform{platformOracle}},
+	{regexp.MustCompile(`\.deb10\.`), []btfPlatform{platformDebian}},
```

Comment:
why only deb10 ?

Reply:
deb11 already has BTF, and debian 9 does not follow this pattern.